x-pack/plugins/security/server/authentication/authentication_service.ts

```diff
@@ -106,8 +106,20 @@ export class AuthenticationService {
     });
 
     http.registerAuth(async (request, response, t) => {
-      // If security is disabled continue with no user credentials and delete the client cookie as well.
+      if (!license.isLicenseAvailable()) {
```

Comment:
__note:__ as a side effect we''l be returning 503 if default distribution of Kibana tries to connect to OSS ES, but I don't think Kibana is functional in such setup at all anyway.